index.html

```diff
@@ -12,14 +12,14 @@
   <body>
     <header class="toolbar">
       <div class="myLogo">My Logo</div>
```

Comment:
- [ ] [OPTIONAL] The logo is also meant to be a link (a tag). This is because is usually clicked on by users to revert back to the homepage. So it would really be nice if it was clickable 😄.

Reply:
Thank you